build.sbt

```diff
@@ -337,6 +328,36 @@ lazy val spark = (project in file("connectors/spark"))
       "net.aichler" % "jupiter-interface" % JupiterKeys.jupiterVersion.value % Test,
       "io.delta" %% "delta-spark" % "4.0.0rc1" % Test,
     ),
+    excludeDependencies ++= Seq(
+      ExclusionRule("com.adobe.testing", "s3mock-junit5")
```

Comment:
what is this for? can you leave a comment? who is bringing in this and why does it need to be excluded